regions/io/ds9/write.py

```diff
@@ -50,10 +59,17 @@ def ds9_objects_to_string(regions, coordsys='fk5', fmt='.4f', radunit='deg'):
         radunitstr = ''
 
     ds9_strings = {
-        'circle': 'circle({x:' + fmt + '},{y:' + fmt + '},{r:' + fmt + '}' + radunitstr + ')\n',
-        'ellipse': 'ellipse({x:' + fmt + '},{y:' + fmt + '},{r1:' + fmt + '}' + radunitstr + ',{r2:' + fmt + '}' + radunitstr + ',{ang:' + fmt + '})\n',
-        'polygon': 'polygon({c})\n',
+        'circle': 'circle({x:' + fmt + '},{y:' + fmt + '},{r:' + fmt + '}' + radunitstr + ')',
```

Comment:
It's not very well-readable in either case, but maybe change these lines to use `.format` to build the string, instead of using + for some and `.format` for point and line below, i.e. have a mix of coding style?
Shortening `.format(f=fmt, r=radunitstr)` might help a little?

Reply:
:+1:, I don't like either version now.  What do you think about this approach?  It's not super pythonic, but I prefer it as a workaround for formatting strings full of {}'s:
```
    ds9_strings = {
        'circle': 'circle({x:FMT},{y:FMT},{r:FMT}RAD)',
        'annulus': 'annulus({x:FMT},{y:FMT},{inner:FMT}RAD,{outer:FMT}RAD)',
        'ellipse': 'ellipse({x:FMT},{y:FMT},{r1:FMT}RAD,{r2:FMT}RAD,{ang:FMT})',
        'rectangle': 'box({x:FMT},{y:FMT},{h1:FMT}RAD,{h2:FMT}RAD,{ang:FMT})',
}
    for key, val in ds9_strings.items():
        ds9_strings[key] = val.replace("FMT", fmt).replace("RAD", radunitstr)
```

Reply:
I mainly dislike the code with "+" as I find it hardest to read. I think your suggestion is best, but I also find the string with many {} OK to read.